.eslintrc

```diff
@@ -1,3 +1,16 @@
 {
-  "extends": "./.config/.eslintrc"
+  "extends": "./.config/.eslintrc",
+  "rules": {
+    // throw error when using global location object, use Grafana's LocationService instead!
+    "no-restricted-globals": ["error", "location"]
+  },
+  "overrides": [
+    {
+      "files": ["src/**/*.test.{ts,tsx}"],
+      // Allow location to be used in test files
+      "rules": {
+        "no-restricted-globals": 0
+      }
+    }
+  ]
```

Comment:
Very cool!
![Screenshot 2025-04-07 at 2 28 19 PM](https://github.com/user-attachments/assets/926ffc08-3771-4510-9f75-b88eb6589b36)

Reply:
> Any tips on how to verify this? When I click around are there sub-paths in the app I can trigger somewhere 🤔

Yeah after deleting your grafana image and running `yarn server` the grafana instance at port 3001 should be in a subpath /grafana

Reply:
To test in another instance you'd need to set GF_SERVER_ROOT_URL and GF_SERVER_SERVE_FROM_SUB_PATH config values: https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#root_url

Reply:
Great idea adding this 👌